data/boosters/ltr-collector-special.yaml

```diff
@@ -42,18 +42,17 @@ sheets:
     count: 101
     foil: true
   showcase_uncommon:
-    count: 84 + 9
+    count: 79 + 9
```

Comment:
@taw the article explicitly mentions 79 cards + 9 nazgul, so this query should hopefully be correct